blazar/manager/service.py

```diff
@@ -369,8 +369,12 @@ def create_lease(self, lease_values):
 
             allocations = self._allocation_candidates(
                 lease_values, reservations)
-            self.enforcement.check_create(
-                context.current(), lease_values, reservations, allocations)
+            try:
+                self.enforcement.check_create(
+                    context.current(), lease_values, reservations, allocations)
+            except common_ex.NotAuthorized as e:
```

Comment:
Do we need the `except` here? What if you  just let it get raised? I would imagine the exception is logged before the HTTP response is returned.

Reply:
@diurnalist I tried without "try-except", but user will see the internal server error, instead of the actual error message.